client/socket.js

```diff
@@ -0,0 +1,5 @@
+import io from 'socket.io-client';
+
+const socket = io('http://localhost:3000');
+
```

Comment:
I think we will need an or statement for what *io* is hooking into here. So an env variable?